Applications/Utils/FileConverter/generateMatPropsFromMatID.cpp

```diff
@@ -69,7 +69,7 @@ int main (int argc, char* argv[])
     std::string const name =
         BaseLib::extractBaseNameWithoutExtension(mesh_arg.getValue());
     // create file
-    std::string const new_matname(name + "prop_");
+    std::string const new_matname(name + "_prop");
```

Comment:
Sorry to reopen it again, but it seems this change is not good.

Reply:
I see that the correction was committed this commit for Process.  It should be committed to 76761fef, which is for Application, where the error was introduced. Anyway, this error is gone, and the file was not touched.

Reply:
Strange. I still see it in your commits. Also if I run git blame locally (or [view](https://github.com/wenqing/ogs/blame/data_member_naming/Applications/Utils/FileConverter/generateMatPropsFromMatID.cpp#L72) on github for that matter) the change is there. You might need to fix the commits and push again.

Reply:
It was changed in the commit for  `Application`, and it was changed back again in the commit for `ProcessLib` (should be changed back in `Application`). Now the change of the file was dropped in the commit for  `Application`, and it dose not appear in commits any more.

Reply:
The implementation in Utils (including file names and paths) are tested with unit test.